.eslintrc.json

```diff
@@ -20,7 +20,8 @@
   },
   "plugins": ["react", "@typescript-eslint", "react-hooks"],
   "rules": {
-    "react-hooks/rules-of-hooks": "warn"
+    "react-hooks/rules-of-hooks": "warn",
+    "@typescript-eslint/no-empty-function": "off"
```

Comment:
Is there any good reason you turned this rule off? It seems to be a good rule to me

Reply:
EditableLabel line 13, I gave an empty object as an initial value to onBlur like this. 
onBlur = {} 
it is my preference not to write onBlur?.() but to use this pattern instead.

Reply:
Interesting. Is this a preferred pattern? Setting default props as `() => {}` is pretty much as the same as passing `undefined` to optional component IMO.